### DISCUSSION TITLE
Occasionally All records are reset after a restart

### DISCUSSION THREAD

**Discussion by EEErink** [seerge/g-helper#1223] [Q&A]
Thank you to the makers for making such a powerful software
but when i restart the computer,it may empty all my settings
how could i avoid this problem?

**Comment by EEErink** [comment]
i found that when i open the G14 2023 without power adapter ,it may reset all settings

**Reply by seerge** [reply]
The only reason it can reset settings, if config.json has broken structure / unreadable by some reason.

**Comment by Maneeha1234SL** [comment]
> Thank you to the makers for making such a powerful software but when i restart the computer,it may empty all my settings how could i avoid this problem?
See if disabling Fast Startup in (Control Panel\Hardware and Sound\Power Options\System Settings) solves the problem
